docs/install/install-with-docker.md

```diff
@@ -151,9 +151,9 @@ docker run -it -v $HOME/.eth2:/data -v /path/to/genesis.ssz:/genesis/genesis.ssz
 ```
 
   </TabItem>
-   <TabItem value="holesky">
+   <TabItem value="hoodi">
 
-Download the Holesky genesis state from [GitHub](https://github.com/eth-clients/holesky/blob/main/metadata/genesis.ssz) to a local file, then run:
+Download the Hoodi genesis state from [GitHub](https://github.com/eth-clients/hoodi/blob/314f727fc019c449905e16c84977415c91e62a30/metadata/genesis.ssz#L2) to a local file, then run:
```

Comment:
we should use the main branch https://github.com/eth-clients/hoodi/blob/main/metadata/genesis.ssz